### ISSUE TITLE
Initial value

### ISSUE THREAD

**Issue by darkdestroyernl** [sag333ar/SRKControls#4]
When i repick the datepicker doesn't remember my selection. Could i easily add this functionality?

**Comment by sag333ar** [comment]
Yes. I know it has that issue. I'm working on it. If you have any suggestion on this or you've code patch for this, feel free to share / contribute.
